lection5/task3.js

```diff
@@ -4,15 +4,12 @@
 
 const str = 'Kala-banga'; //слово
 const vowelsStr = 'aeiouy'; //гласные
-const consonantsStr = 'bcdfghjklmnpqrstvwxyz'; //гласные
+const consonantsStr = 'bcdfghjklmnpqrstvwxyz'; //согласные
 
 let vowels = 0;
 let consonants = 0;
 
 for (let char of str) {
```

Comment:
```suggestion
for (let char of str.toLowerCase()) {
```
Convert test word to lower case